src/components/Header/Index.module.css

```diff
@@ -0,0 +1,26 @@
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.header {
+  display: grid;
+  grid-template-columns: 15% 1fr;
+  grid-template-rows: 100px;
```

Comment:
smth in %, smth in px. I think you need to think about standards! For my opinion better use em